CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # Changelog
 
+## v0.10.0 (2016-11-2)
+
+### Enhancements
+
+* **Snapshot API shorthands. ([#771](https://github.com/DevExpress/testcafe/issues/771))**
+  
+  Previously, if you needed to use a single property from the snapshot, you had to write two assignments
+
+  ```js
+  const snapshot = await selector();
+  const nodeType = snapshot.nodeType;
+  ```
+
+  or additional parentheses.
+
+  ```js
+  const nodeType = (await selector()).nodeType;
+  ```
+
+  Now snapshot methods and property getters are exposed by selectors
+  (and by promises returned by selectors as well) so that you can write more compact code.
+
+  ```js
+  const nodeType = await selector.nodeType;
+
+  // or
+
+  const nodeType = await selector('someParam').nodeType;
+  ```
+
+  However, shorthand properties are not enough to avoid using parentheses with dictionary properties
+  like `style`, `attributes` or `boudingClientRect`.
+  
+  ```js
+  const width = (await selector.style)['width'];
+  ```
+
+  That is why we have also introduced shorthand methods for these dictionaries: `getStyleProperty`, `getAttribute` and `getBoundingClientRectProperty`.
+
+  ```js
+  const width = await selector.getStyleProperty('width');
+  const id    = await selector.getAttribute('id');
+  const left  = await selector.getBoundingClientRectProperty('left');
+  ```
+
+  Finally, we have added the `hasClass` method.
+
+  ```js
+  if (await selector.hasClass('foo')) {
+      //...
+  }
+  ```
+
+  See [Snapshot API Shorthands](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#snapshot-api-shorthands).  
+* **Improved automatic wait mechanism. ([#245](https://github.com/DevExpress/testcafe/issues/245))**
+
+  We got rid of unnecessary waiting and tests now run almost two times faster.
+
+  ![Tests running in v0.10.0 vs v0.9.0](https://raw.githubusercontent.com/DevExpress/testcafe/master/media/new-0-10-0-autowait.gif)
+* **Test execution speed control. ([#938](https://github.com/DevExpress/testcafe/issues/938))**
+
+  We have introduced an option that allows you to specify how fast tests run.
+  It controls the cursor speed and the delay between actions.
```

Comment:
In fact it just adds delay between actions and don't change cursor behavior. 
Action speed (it's not implemented yet, see https://github.com/DevExpress/testcafe/issues/865) will have affect on cursor speed